packages/next/next.config.js

```diff
@@ -1,6 +1,11 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  env: {
+    pinataApiKey: '46df177599dba690a737',
+    pinataApiSecret:
+      '16beed50084ea86097af3852d40ded8664621f274a60a046b3335cf9447f545b',
```

Comment:
Should they set their own key here?

Reply:
Agreed that for this test UI, the way it is setup is not a problem. The pinata API keys are part of a free account.